daml-lf/spec/transaction.rst

```diff
@@ -155,6 +155,8 @@ This table lists every version of this specification in ascending order
 +--------------------+-----------------+
 |                  5 |      2019-03-12 |
 +--------------------+-----------------+
+|                  6 |      2019-04-29 |
++--------------------+-----------------+
```

Comment:
There's also a version reference at top of file.

Reply:
done.